Docs/source/usage/parameters.rst

```diff
@@ -228,6 +228,16 @@ Overall simulation parameters
         \boldsymbol{\nabla}^2 \phi = - \rho/\epsilon_0 \qquad \boldsymbol{E} = - \boldsymbol{\nabla}\phi \\
         \boldsymbol{\nabla}^2 \boldsymbol{A} = - \mu_0 \boldsymbol{j} \qquad \boldsymbol{B} = \boldsymbol{\nabla}\times\boldsymbol{A}
 
+    * ``labframe-semi-implicit``: Poisson's equation is solved with a modified dielectric function
+      to create a semi-implicit scheme in which grid heating effects due to unresolved
+      plasma modes are suppressed. If this option is used the additional parameter
+      ``warpx.semi_implicit_factor`` can also be specified to set the value of :math:`C_{SI}`.
```

Comment:
We can mention the default value for `semi_implicit_factor` here.

The simulation will be stable if `C_{SI} \geq 1` + mention disadvantages of setting it too high.